code/modules/surgery/organs_internal.dm

```diff
@@ -36,6 +35,16 @@
 			return 0
 		return 1
 
+/datum/surgery/organ_manipulation_boneless/can_start(mob/user, mob/living/carbon/target)
+	if(istype(target,/mob/living/carbon/human))
+		var/mob/living/carbon/human/H = target
+		var/obj/item/organ/external/affected = H.get_organ(user.zone_sel.selecting)
+		if(affected && (affected.status & ORGAN_ROBOT))
+			return 0
+		if(target.get_species() == "Diona")
+			return 1
+		return 0
```

Comment:
This should probably check for the `encased` variable on `affected`, rather than the species of the owner - that way, you can slap a human head on a diona and have it still work